pyspline/pySpline.py

```diff
@@ -6,37 +6,18 @@
 :class:`Volume`
 """
 
-# Standard Python modules
-import warnings
-
 # External modules
-import numpy
+import numpy as np
 
 # Local modules
 from .pyCurve import Curve
 from .pySurface import Surface
 from .pyVolume import Volume
 from .utils import Error
 
-
-# For backwards compatibility, the old curve, surface and volume definitions:
-def curve(*args, **kwargs):
-    warnings.warn("pySpline.curve has been changed to Curve()")
-    return Curve(*args, **kwargs)
-
-
-def surface(*args, **kwargs):
-    warnings.warn("pySpline.surface has been changed to Surface()")
-    return Surface(*args, **kwargs)
-
-
-def volume(*args, **kwargs):
-    warnings.warn("pySpline.volume has been changed to Volume()")
-    return Volume(*args, **kwargs)
```

Comment:
I am not familiar with the use-cases for these definitions or what backward-compatibility removing them breaks, but are we certain that this will not cause problems in our workflow? I assume it will break run scripts that specifically invoke `pyspline.<object>`, but not anything critical in our codes. I want to flag it to be extra certain.

Reply:
I did some grep over our code and didn't find anything. Certainly I don't think anything that we use often relies on this, since I don't recall _ever_ seeing that warning.